index.html

```diff
@@ -4,10 +4,217 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="mobile-version-style.css">
+    <title>My Portfolio Mobile Version</title>
+    <link rel="shortcut icon" href="Images/favicon.ico"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+   <header class="presentation">
+      <nav class="nav-container">
+         <img class="my-logo" src="Images/Logo HG.png">
+         <i style="color: #acb7c3" class="fa fa-bars"></i>
```

Comment:
- [Optional] Kindly consider using an indentation of two spaces in all your documents. This is a convention and also best practice.